cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryReader.java

```diff
@@ -27,9 +27,15 @@ public Polygon readPolygon(String name, int index) {
 		Variable polyvar = ds.findVariable(name);
 		if(polyvar == null) return null;
 		
-		// create a blank cf Polygon
-		Polygon poly = new CFPolygon();
-		return poly.setupPolygon(ds, polyvar, index);
+		Polygon poly = null;
+		
+		// CFConvention
+		if(ds.findGlobalAttribute(CF.CONVENTIONS) != null)
+			if(ucar.nc2.dataset.conv.CF1Convention.getVersion(ds.findGlobalAttribute(CF.CONVENTIONS).getStringValue()) >= 0)

```

Comment:
Based on my read, `CF1Convention.getVersion()` is delegating to `CF1Convention.extractVersion()`, which looks like it's returning the minor version of the CF convention. By "minor version" I mean the version after the decimal, as opposed to the major version which comes before the decimal.

CF support for Simple Geometries begins to be supported in CF 1.8.

Is zero the right value to compare against to condition the creation of CF 1.8-specific functionality?

Reply:
Nice catch, originally it was to allow for any CF-1 Convention, but CF-1.8+ are the only ones with simple geometry support. Will be changed to check for 8!